app/src/Widget/List.jsx

```diff
@@ -72,6 +72,17 @@ const useStyles = (theme) => ({
       },
     }
   },
+  // Add some lateral space for the slider on larger displays
+  sliderRoot: {
+    [theme.breakpoints.up('md')]: {
+      marginLeft: theme.spacing(4),
+      marginRight: theme.spacing(4),
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginLeft: theme.spacing(1),
+      marginRight: theme.spacing(1),
```

Comment:
I've tested with just `1` and it overlaps with lateral buttons. Looks like the minimal for small devices should be `3`.